rest/model/task_annotations.go

```diff
@@ -155,3 +161,28 @@ func ArrAPIIssueLinkArrtaskannotationsIssueLink(t []APIIssueLink) []annotations.
 	}
 	return m
 }
+
+func GetJiraTickets(issuelinks []APIIssueLink) ([]APIIssueLink, error) {
+	settings := evergreen.GetEnvironment().Settings()
+	jiraHandler := thirdparty.NewJiraHandler(*settings.Jira.Export())
+	catcher := grip.NewBasicCatcher()
+
+	var res []APIIssueLink
+	for _, issue := range issuelinks {
+		urlObject, err := url.Parse(*issue.URL)
+		if err != nil {
+			catcher.Add(errors.Wrap(err, "problem parsing issue string"))
+		}
+		if urlObject.Host == "jira.mongodb.org" {
```

Comment:
You'll also want to check that urlObject isn't nil (which it would be if an error was hit)